torch/nested/__init__.py

```diff
@@ -137,4 +113,34 @@ def as_nested_tensor(tensor_list: List[Tensor], dtype: Optional[DType] = None, d
     >>> pt_small = torch.nested.to_padded_tensor(nt, 2.0, (2, 2, 2))
     RuntimeError: Value in output_size is less than NestedTensor padded size. Truncation is not supported.
 
-""")
+""",
+)
+
+nested_tensor = _add_docstr(
+    _nested.nested_tensor,
+    r"""
```

Comment:
nit: when using `_add_docstr`, in order to render the arguments for the function a line containing the signature of the function needs to be added here (see `to_padded_tensor` above for an example)

(the preview for the docs is available under helpful links [in the pytorch-bot comment on this PR](https://github.com/pytorch/pytorch/pull/88213#issuecomment-1298866390) if you would like to look at the rendering!)